packages/fx-core/resource/package.nls.json

```diff
@@ -1151,6 +1151,7 @@
   "driver.deploy.error.unknownError": "Unknown error happened: %s",
   "driver.deploy.notice.deployAcceleration": "Deploying to Azure App Service takes a long time. Consider referring to this document to optimize your deployment:",
   "driver.deploy.error.noFileFindInDistributionFolder": "No file is found in distribution folder. Please check if the distribution folder is empty or you have ignored everything.",
+  "driver.deploy.error.deployPrepareComplete": "Deploy prepare complete. Cannot deploy to a remote server until the correct resourceId is provided.",
```

Comment:
Wording Nit: `prepare` is a verb, so I think it's inpropriate to use it here. Maybe something like `Preparations of deployment are complete`?

Reply:
Nic catch!